packages/components/src/components/overflow-menu/overflow-menu.js

```diff
@@ -114,6 +115,12 @@ class OverflowMenu extends mixin(
    */
   constructor(element, options) {
     super(element, options);
+
+    if (this.element.getAttribute('role') !== 'button') {
+      // Would prefer to use the aria-controls with a specific ID but we
+      // don't have the menuOptions list at this point to pull the ID from
+      this.trigger = this.element.querySelector(this.options.selectorTrigger);
```

Comment:
We tend to add ...`Node` suffix for this kind of instance properties. This case, `triggerNode`.